dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/QdrantVectorDb.cs

```diff
@@ -0,0 +1,221 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel.Memory.Storage;
+using Qdrant.DotNet;
+
+
+namespace Microsoft.SemanticKernel.Skills.Memory.VectorDB;
+
+/// <summary>
+/// An implementation of a client for the Qdrant VectorDB. This class is used to 
+/// connect, create, delete, and get embeddings data from a Qdrant VectorDB instance.
+/// </summary>
+
+public class QdrantVectorDb
```

Comment:
I would like to see this renamed to QdrantVectorDBClient to indicate that this wraps the Qdrant API